src/goose/client.clj

```diff
@@ -45,3 +60,11 @@
   `args` are variadic."
   [opts sec execute-fn-sym & args]
   (enqueue opts (u/add-sec sec) execute-fn-sym args))
+
+(defn perform-every
```

Comment:
An API namespace to manage periodic jobs is necessary. Just CRD operations like enqueued/scheduled jobs should suffice

Reply:
Done